### PR TITLE
[v1.x] Fix cu100 pip link

### PR DESCRIPTION
**Pull request by waytrue17**
## Description ##
Fix cu100 pip link

### Essentials ###
- [ ] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Review by szha — APPROVED** [review]

File: tools/pip/doc/CU101_ADDITIONAL.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ This package supports Linux and Windows platforms. You may also want to check:
 - [mxnet-cu112](https://pypi.python.org/pypi/mxnet-cu112/) with CUDA-11.2 support.
 - [mxnet-cu110](https://pypi.python.org/pypi/mxnet-cu110/) with CUDA-11.0 support.
 - [mxnet-cu102](https://pypi.python.org/pypi/mxnet-cu102/) with CUDA-10.2 support.
-- [mxnet-cu100](https://pypi.python.org/pypi/mxnet-cu101/) with CUDA-10.0 support.
```
Comment (samskalicky) [inline tools/pip/doc/CU101_ADDITIONAL.md:24]: do we want to list the mxnet-cu101 variant too?

**Review by samskalicky — COMMENTED** [review]
Can we try and fix the formatting so that the list shows up correctly? It looks ok here:
https://github.com/apache/incubator-mxnet/blob/ea66f6cd6e0f0641aa5054ecc0d31b4a88b3d6d4/tools/pip/doc/CU101_ADDITIONAL.md
but then turns into a jumbled paragraph here: https://pypi.org/project/mxnet-cu101/

File: tools/pip/doc/CU101_ADDITIONAL.md (hunk 1 above)
Comment (waytrue17) [reply tools/pip/doc/CU101_ADDITIONAL.md:24]: Seems this part is a reference to other variants not including itself. Other files like `tools/pip/doc/CU100_ADDITIONAL.md` and `tools/pip/doc/CU102_ADDITIONAL.md` do not include the self link either

File: tools/pip/doc/CU101_ADDITIONAL.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ This package supports Linux and Windows platforms. You may also want to check:
 - [mxnet-cu112](https://pypi.python.org/pypi/mxnet-cu112/) with CUDA-11.2 support.
```
Comment (samskalicky) [inline tools/pip/doc/CU101_ADDITIONAL.md:21]: can we add a newline here between the text and the list?

File: tools/pip/doc/CU101_ADDITIONAL.md (hunk 1 above)
Comment (samskalicky) [reply tools/pip/doc/CU101_ADDITIONAL.md:24]: got it, makes sense

File: tools/pip/doc/CU101_ADDITIONAL.md (hunk 2 above)
Comment (waytrue17) [reply tools/pip/doc/CU101_ADDITIONAL.md:22]: Added new lines in all variant files. Thanks for the suggestion!

**Review by samskalicky — APPROVED** [review]
Thanks @waytrue17 !

**Pull request merged**